<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1467</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YapiBot Commander</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="VideoView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Forw">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>100</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Forw</string>
    </property>
    <property name="shortcut">
     <string>Z</string>
    </property>
    <property name="autoRepeatDelay">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Rear">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>160</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Rear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Left">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Right">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="TouchPad" name="touchPad" native="true">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>220</y>
      <width>199</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="CameraTild">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>130</y>
      <width>29</width>
      <height>160</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="heading">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>0</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>3</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="speedRight">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>0</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="speedLeft">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>0</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="range">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="measLeft">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>100</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="measRight">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>100</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CompassCalib">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Compass Cal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveStraight">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>move straight</string>
    </property>
   </widget>
   <widget class="QPushButton" name="alignBearing">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>500</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>align bearing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveBearing">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>500</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>move bearing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>500</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="script">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>500</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>script</string>
    </property>
   </widget>
   <widget class="QDial" name="speed">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>140</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ConsSpeed">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QComboBox" name="paramlist">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>440</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="paramval">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>440</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="Map" name="uiMap" native="true">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>50</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshMap">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>500</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh Map</string>
    </property>
   </widget>
   <widget class="QwtCompass" name="Compass">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="frameShadow">
     <enum>QwtDial::Sunken</enum>
    </property>
    <property name="mode">
     <enum>QwtDial::RotateNeedle</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1467</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtCompass</class>
   <extends>QwtDial</extends>
   <header>qwt_compass.h</header>
  </customwidget>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
  <customwidget>
   <class>TouchPad</class>
   <extends>QWidget</extends>
   <header>touchpad.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Map</class>
   <extends>QWidget</extends>
   <header>map.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
